pkg/tracker/disputeChecker.go

```diff
@@ -47,11 +47,11 @@ type ValueCheckResult struct {
 // CheckValueAtTime queries for the details regarding the disputed value.
 func CheckValueAtTime(cfg *config.Config, reqID uint64, val *big.Int, at time.Time) (*ValueCheckResult, error) {
 
-	// check the value in 5 places, spread over cfg.DisputeTimeDelta.Duration.
+	// check the value in 5 places, spread over cfg.Trackers.DisputeTimeDelta.Duration.
```

Comment:
If we are changing comments lets also fix them to follow the golang standards :smiley: 
```suggestion
	// Check the value in 5 places, spread over cfg.Trackers.DisputeTimeDelta.Duration.
```

Reply:
noted :)